plugins/inputs/win_eventlog/README.md

```diff
@@ -11,6 +11,8 @@ Supports Windows Vista and higher.
   xpath_query = "Event/System[EventID=999]"
 ```
 
+Alternatively it is possible to use special XML query documented [here](https://docs.microsoft.com/en-us/windows/win32/wes/consuming-events), in this case eventlog_name field will be ignored (use triple quote for xml in telegraf config file).
+
```

Comment:
Can you please give an example for how to use this after ignoring eventlog_name

Reply:
@anuar45 : Is there any possibility to have filter on the basis of Event Data or message of the event log. 
Also can you please help with an example for below:
Alternatively it is possible to use special XML query documented [here](https://docs.microsoft.com/en-us/windows/win32/wes/consuming-events), in this case eventlog_name field will be ignored (use triple quote for xml in telegraf config file).